custom/icds_reports/tasks.py

```diff
@@ -1209,6 +1249,114 @@ def build_incentive_report(agg_date=None):
                 build_incentive_files.delay(location, agg_date, file_format, 3, location.parent.parent, location.parent)
 
 
+# performance row is AWCIncentiveReport object with 'awc_id', 'is_launched', 'valid_visits',
+# 'visit_denominator', 'wer_weighed', 'wer_eligible', 'incentive_eligible'
+# awcagg_row is AggAwc object with 'awc_id', 'is_launched', 'valid_visits', 'expected_visits'
+def get_awc_is_launched_mismatch_row(performance_row, awcagg_row):
+    is_launched_from_awc = False
+    if awcagg_row[1].lower() == 'yes':
+        is_launched_from_awc = True
+    # checking if is_launched from incentive report is same as that of aggregate
+    if performance_row[1] != is_launched_from_awc:
+        row_data = [performance_row[0], awcagg_row[0], performance_row[1], is_launched_from_awc]
```

Comment:
It looks first value in the row is just the awc_id, then  no need to include both `performance_row[0]` and  `awcagg_row[0]`.

Reply:
since the checks are relying on the order of awc_ids, i thought to add ids from both just in case if there is a mismatch, we can eliminate this dilemma